pom.xml

```diff
@@ -43,13 +43,14 @@
         <lombok.version>1.18.12</lombok.version>
         <lombok-maven-plugin.version>1.18.10.0</lombok-maven-plugin.version>
         <delombok.output>target/generated-sources-for-javadocs</delombok.output>
+        <!-- TODO: Kotlin 1.4 - https://github.com/slackapi/java-slack-sdk/issues/545 -->
         <kotlin.version>1.3.72</kotlin.version>
         <slf4j.version>1.7.30</slf4j.version>
-        <aws.s3.version>1.11.836</aws.s3.version>
+        <aws.s3.version>1.11.846</aws.s3.version>
```

Comment:
This is the only dependency for the SDK. This is a totally optional one (not compile scope but provided scope) and developers just use the latest patch versions.